### PR TITLE
Add Filter

### PR DESCRIPTION
**Pull request by yujin0131**
Add Filter

### CODE REVIEW

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java [hunk 1]
```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
...
+
+			outer : for(int i = 0; i < ListArr.length; i++) {
+				
+				if(ListArr[i].equals("0") && ListArr[i+1].equals("0")) {// 둘다 없을때
+					query = "";
+					break outer ;
+				}else if(!(ListArr[0].equals("0")) && !(ListArr[i].equals("0")) && ListArr[i+1] == null) { //fac, ame 다 있는 마지막
+					query += ListArr[i] + queryLast;
+					System.out.println("다있는 마지막 " + query);
+					break outer;
```
Comment (yujin0131) [inline Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:154]: When there are both filters in param

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java [hunk 2]
```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
...
+				}else if(ListArr[i].equals("0") && !(ListArr[i+1].equals("0"))) { //ame이 없을 i = 0 일 때 넘겨주는 코드
+					System.out.println("ame 없는 처음 " + query);
+					continue outer;
```
Comment (yujin0131) [inline Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:157]: First time if  there is facility in param and there is no amenity

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java [hunk 3]
```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
...
+				}else if(ListArr[0].equals("0") && !(ListArr[i].equals("0")) && ListArr[i+1] == null ) { //ame이 없고 facil이 마지막
+					query += ListArr[i] + queryLast;
+					System.out.println("ame없는 마지막 " + query);
+					break outer;
```
Comment (yujin0131) [inline Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:161]: When last query with facility in the param and no amenity

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java [hunk 4]
```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
...
+				}else if(ListArr[i] != null && ListArr[i+1].equals("0") && ListArr[i+2] == null ) { //facilitylist가 없을때
+					query += ListArr[i] + queryLast;
+					System.out.println("facility없는 마지막 " + query);
+					break outer;
```
Comment (yujin0131) [inline Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:165]: When last query with amenity in the param and no facility

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java [hunk 5]
```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
...
+				}else{//ame만 있거나 둘다 많이씩 있을때
+					query += ListArr[i] + queryMiddle;
+					System.out.println("중간 : " + query);
```
Comment (yujin0131) [inline Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:168]: Else everything

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java [hunk 6]
```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
...
+			String queryFirst = " and (facility like '%";
+			String queryMiddle = "%' or facility like'%";
+			String queryLast = "%')";
+			String query = queryFirst;
+
+			outer : for(int i = 0; i < ListArr.length; i++) {
+				
+				if(ListArr[i].equals("0") && ListArr[i+1].equals("0")) {// 둘다 없을때
+					query = "";
+					break outer ;
```
Comment (yujin0131) [inline Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:150]: when nothing

File: Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java (hunk 1 above)
Comment (yoonwooseong) [reply Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java:154]: It is good to delete "system.out.println" because it is slow 👍

**Pull request merged**